src/lib.rs

```diff
@@ -7,24 +7,33 @@ use std::sync::Arc;
 
 use wasm_bindgen::prelude::*;
 
-#[wasm_bindgen]
+#[wasm_bindgen(module = "/public/qr-scanner-worker.min.js")]
+extern "C" {
+    #[derive(Clone, Debug)]
+    type QrWorker;
+
+    #[wasm_bindgen(method, js_name = createWorker)]
+    fn createWorker(this: &QrWorker);
+}
+
+#[wasm_bindgen(module = "/public/qr-scanner-wrapper.min.js")]
```

Comment:
:heart_eyes:

Reply:
yeah this one took me a bit to figure out :)